### DISCUSSION TITLE
How do I choose where to place Table of Contents (toc: true)?

### DISCUSSION THREAD

**Discussion by cowvin0** [quarto-dev/quarto-cli#7389] [Q&A] [answered]
### Description

Hey there. I'm adding a table of contents in a Quarto document with toc: true, but it only appears at the beginning of the page. How could I choose where to place it? I'm using PDF format.

**Comment by mcanouil** [comment, marked as answer]
See partials/template: https://quarto.org/docs/journals/templates.html#latex-partials

**Reply by josephsalmon** [reply]
Any alternative for TOC with revealjs in Quarto?

The proposition given in this context by

https://stackoverflow.com/questions/75024777/changing-on-which-slide-the-toc-appears-in-revealjs-slides-using-quarto

does not interplay well with the menu unfortunately.

**Reply by mcanouil** [reply]
It probably needs to be adapted.
